client/my-sites/plans-features-main/style.scss

```diff
@@ -68,7 +68,6 @@ $plan-features-header-banner-height: 20px;
 .calypso-notice.plan-features-main__notice {
 	margin: 0 20px 24px;
 	width: unset;
-	text-wrap: pretty;
```

Comment:
No longer needed because this is a wrapper for `calypso-notice__content` which is now a `p` tag.